### ISSUE TITLE
multiple maps with the same PCA vectors

### ISSUE THREAD

**Issue by bernstei** [BingqingCheng/ASAP#52]
It would be useful to do maps of multiple data sets in the same space (e.g. same PCA vectors), side by side (multiple panels on the same page), colored by some property.

**Comment by BingqingCheng** [comment]
So you mean PCA for a big dataset that is a collection of subsets, and then plot the PCA of the subsets on different panels?

**Comment by bernstei** [comment]
> So you mean PCA for a big dataset that is a collection of subsets, and then plot the PCA of the subsets on different panels?


Yes, I think so.  Basically, I want to compare sets of configurations on a low-D map, colored by some quantity.  If I just want to compare the low-D distribution I can use the color to indicate which set they come from, but if I want the color to be some other quantity, I need separate panels (but all using the same PCA vectors and same axes).
